### PR TITLE
Create money8203.json

### PR DESCRIPTION
**Pull request by money8203**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1834"></a>

### CODE REVIEW

**Review by money8203 — COMMENTED** [review]
Updated money8203.json

**Review by kumarsonsoff3 — APPROVED** [review]
Look good to me! 🎉

File: public/data/money8203.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Manish Tyagi",
+  "type": "personal",
+  "bio": "Techology Optimist | Open Source Enthusiast",
```
Comment (kabir0x23) [inline public/data/money8203.json:4]:
```suggestion
  "bio": "Technology Optimist | Open Source Enthusiast",
```

**Review by kabir0x23 — CHANGES_REQUESTED** [review]
Spelling mistake: Techology > Technology.
Rest everything is set 😄.

File: public/data/money8203.json (hunk 1 above)
Comment (money8203) [reply public/data/money8203.json:4]: @kabir0x23  Thanks for clearly reviewing my code

**Review by kabir0x23 — APPROVED** [review]
Looks good to me ✔.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks for contributing! 🎉

**Comment by money8203** [comment]
@Panquesito7 Always happy to contribute !

**Pull request merged**